src/common/hist_util.h

```diff
@@ -43,6 +44,10 @@ struct GHistEntry {
     sum_hess += e.sum_hess;
   }
 
+  inline static void Reduce(GHistEntry& a, const GHistEntry& b) { // NOLINT(*)
```

Comment:
The GHistEntry data structure appears to be just a duplication of GradStats which already has the Reduce method. Maybe this is a question from other PR but I think these should be combined.

Reply:
@RAMitchell There are some changes in #3825 .

Reply:
I will leave the changes to #3825